pravega/src/main/java/akka/stream/alpakka/pravega/javadsl/Pravega.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.pravega.javadsl;
+
+import akka.Done;
+import akka.NotUsed;
+import akka.stream.alpakka.pravega.PravegaEvent;
+import akka.stream.alpakka.pravega.WriterSettings;
+import akka.stream.javadsl.Flow;
+import akka.stream.javadsl.Keep;
+import akka.stream.javadsl.Sink;
+import akka.stream.javadsl.Source;
+
+import io.pravega.client.stream.ReaderConfig;
+import scala.compat.java8.FutureConverters;
+import static scala.compat.java8.OptionConverters.toScala;
+
+import java.util.Optional;
+import java.util.concurrent.CompletionStage;
+import java.util.function.Function;
+
+import akka.stream.alpakka.pravega.impl.PravegaFlow;
+import akka.stream.alpakka.pravega.impl.PravegaSource;
+
+import akka.stream.alpakka.pravega.ReaderSettings;
+
+import io.pravega.client.ClientConfig;
+import io.pravega.client.stream.EventStreamWriter;
+import io.pravega.client.stream.EventWriterConfig;
+import io.pravega.client.stream.Serializer;
+import scala.compat.java8.Priority1FunctionConverters;
+
+public class Pravega {
+
+  /**
+   * Messages are read from a Pravega stream.
+   *
+   * <p>Materialized value is a future which completes to Done as soon as Pravega reader in open.
+   */
+  public static <V> Source<PravegaEvent<V>, CompletionStage<Done>> source(
+      String scope, String topic, ReaderSettings<V> settings) {
```

Comment:
I'm not familiar enough with all the Pravega terminology but shouldn't it be `stream` instead o `topic`? 

```suggestion
      String scope, String streamName, ReaderSettings<V> readerSettings) {
```